modules/access-service.adoc

```diff
@@ -6,18 +6,15 @@
 
 = Accessing installed services on your cluster
 
-// Should this be the OCM instead of "OpenShift console"?
 After you successfully install a service on your cluster, you can access the service through the OpenShift console.
 
-
 .Prerequisites
 
 * You have successfully installed a service on your {product-title} cluster.
 
-
 .Procedure
 
-. Navigate to the *Clusters* page in link:https://cloud.redhat.com/openshift/[OpenShift Cluster Manager (OCM)].
+. Navigate to the *Clusters* page in link:https://cloud.redhat.com/openshift/[OpenShift Cluster Manager].
```

Comment:
This link should be https://console.redhat.com/openshift/ (the site changed in last year, so not sure how long redirects will continue working)